spec/car_spec.rb

```diff
@@ -3,38 +3,52 @@
 
 RSpec.describe Car do
   it 'exists' do
-    car = Car.new("Ford Mustang", 1500, 36)
+    car = Car.new("Ford", "Mustang", 1500, 36)
```

Comment:
You wouldn't want to change your tests by adding in the `,` between the string `"Ford Mustang"`. That will take place in your class.